Documentation/Functions/Typolink.rst

```diff
@@ -728,65 +728,4 @@ Aspects `identifier` and `uid` are mandatory for this link handler.
 Using link handlers
 ===================
 
-A feature allows you to register a link handler
-for a keyword you define. For example, you can link to a page with id
-34 with "<link 34>" in a typical bodytext field which converts <link>
-tags with "->typolink". But what if you have an extension,
-"pressrelease", and wanted to link to a press release item displayed
-by a plugin on some page you don't remember? With this feature it's
-possible to create the logic for this in that extension.
-
-So, in a link field (the "parameter" value for ->typolink) you could
-enter "pressrelease:123":
-
-.. figure:: ../Images/LinkHandler.png
-   :alt: Screenshot of the "link" field in the TYPO3 Backend.
-
-Some TypoScript will usually transfer this value to the "parameter"
-attribute of the ->typolink call. When "pressrelease:123" enters
-->typolink as the "parameter" it will be checked if "pressrelease" is
-a keyword with which a link handler is associated and if so, that
-handler is allowed to create the link.
-
-Registering the handler for keyword "pressrelease" is done like this:
-
-.. code-block:: php
-
-   $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['tslib/class.tslib_content.php']
-     ['typolinkLinkHandler']['pressrelease'] =
-     'EXT:pressrelease/class.linkHandler.php:&tx_linkHandler';
-
-The class file :file:`pressrelease/class.linkHandler.php` contains the class
-:php:`tx_linkHandler` which could look like this:
-
-.. code-block:: php
-
-   class tx_linkHandler {
-      function main($linktxt, $conf, $linkHandlerKeyword,
-         $linkHandlerValue, $link_param, &$pObj) {
-         $lconf = array();
-         $lconf['useCacheHash'] = 1;
-         $lconf['parameter'] = 34;
-         $lconf['additionalParams'] = '&tx_pressrelease[showUid]=' .
-            rawurlencode($linkHandlerValue);
-         return $pObj->typoLink($linktxt, $lconf);
-      }
-   }
-
-In this function, the value part after the keyword is set as the value
-of a GET parameter, `&tx_pressrelease[showUid]` and the "parameter"
-value of a new ->typolink call is set to "34" which assumes that on
-page ID 34 a plugin is put that will display pressrelease 123 when
-called with `&tx_pressrelease[showUid]=123`. In addition you can see
-the "useCacheHash" attribute for the typolink function used in order
-to produce a cached display.
-
-The link that results from this operation will look like this:
-
-.. code-block:: html
-
-   <a href="index.php?id=34&amp;
-     tx_pressrelease[showUid]=123%3A456&amp;cHash=c0551fead6" >
-
-The link would be encoded with RealURL and respect :ts:`config.linkVars`
-as long as ->typolink is used to generate the final URL.
+see :ref:`Link Handler Documentation in TYPO3 Explained<t3coreapi:linkhandler>`
```

Comment:
① I would make that a sentence: ```suggestion
See :ref:`Link handler documentation in "TYPO3 Explained" <t3coreapi:linkhandler>`.
```
② The blank before < was missing and would be an error
③ I think we agreed on using lower case wherever possible.